arm/cognitiveservices/accounts.go

```diff
@@ -61,7 +61,7 @@ func (client AccountsClient) Create(resourceGroupName string, accountName string
 			Constraints: []validation.Constraint{{Target: "parameters.Sku", Name: validation.Null, Rule: true,
 				Chain: []validation.Constraint{{Target: "parameters.Sku.Tier", Name: validation.ReadOnly, Rule: true, Chain: nil}}},
 				{Target: "parameters.Location", Name: validation.Null, Rule: true, Chain: nil},
-				{Target: "parameters.Properties", Name: validation.Null, Rule: true, Chain: nil}}}}); err != nil {
+				{Target: "parameters.map[string]interface{}", Name: validation.Null, Rule: true, Chain: nil}}}}); err != nil {
```

Comment:
I wonder if this actually works? Maybe the problem comes from a bad defined swagger?